.github/workflows/ci.yml

```diff
@@ -1,17 +1,81 @@
 name: CI
 
-on: [push]
+on: push
 
 jobs:
-  build:
+  webknossos_linux:
     runs-on: ubuntu-latest
     strategy:
       max-parallel: 4
       matrix:
         python-version: [3.7, 3.8]
+    defaults:
+      run:
+        working-directory: webknossos
 
     steps:
     - uses: actions/checkout@v2
+      with:
+        fetch-depth: 2
+    - uses: marceloprado/has-changed-path@v1
+      id: changed
+      with:
+        paths: webknossos
```

Comment:
This just compares the last two commits, but earlier runs may have failed already, and a new commit without changes in the code would make the CI green. I'll exchange this with https://github.com/RasaHQ/pr-changed-files-filter therefore.